fml/platform/fuchsia/message_loop_fuchsia.h

```diff
@@ -24,8 +26,13 @@ class MessageLoopFuchsia : public MessageLoopImpl {
 
   void WakeUp(fml::TimePoint time_point) override;
 
+  zx_handle_t timer_;
+
+  async::Wait* timer_wait_;
   async::Loop loop_;
 
+  bool running_;
```

Comment:
`std::atomic_bool` as it will be set from different threads.

Reply:
do we need to change the other MessageLoopImpls to be atomic_bool as well?

Reply:
Yeah